### PR TITLE
Fixed double ";" when using putValue on a line with a comment

### PR DESCRIPTION
**Pull request by louiswouters**
Refactored the code to use a join so that it doesn't need to check for adding spaces everytime.
We use a join at the end so that every element is separated by a space.

It also fixes the bug where it would grab the comment including ";" and then add another ";" when reinserting the comment

### CODE REVIEW

**Comment by louiswouters** [comment]
Line 173-175 could possible be removed as well, since I don't think there are other parameters then G M T S F X Y Z E.

File: plugins/PostProcessingPlugin/Script.py [hunk 1]
```diff
@@ -141,52 +141,45 @@ def putValue(self, line: str = "", **kwargs) -> str:
...
+        Any other parameters will be added in arbitrary order.
 
         :param line: The original g-code line that must be modified. If not
             provided, an entirely new g-code line will be produced.
         :return: A line of g-code with the desired parameters filled in.
         """
-
-        #Strip the comment.
-        comment = ""
+        # Strip the comment.
```
Comment (nallath) [inline plugins/PostProcessingPlugin/Script.py:151]: Uh, how does that change the behavior? As far as I can see, the changed code will do exactly the same.

**Comment by louiswouters** [comment]
PutValue( "G92 X45 ; move", Y=12) would evaluate to G92 X45 Y12 ;; move. This fixes the double semicolon

**Comment by ninovanhooff** [comment]
Assuming the code actually works, I'd say this is an improvement. The kwargs.pop() is easy to gloss over though, please put it on a separate line so it is more clear that the prioritized kwargs will not be iterated a second time in the other for loop

**Pull request merged**